core/templates/dev/head/pages/exploration_editor/RouterServices.js

```diff
@@ -85,6 +85,17 @@ oppia.factory('routerService', [
       }
     });
 
+    // Select2 dropdowns does not automatically get populated
+    // when the tab is loaded, so we have to force it to refresh.
+    // Similar issue happened in Preferences.js and the same bug
```

Comment:
Also, comments should be phrased in terms of what makes sense to future devs looking at the future version of the repository. So, "This follows a similar approach in Preferences.js" might work, since it's describing the code as-is. But talking about what bugs were fixed in the past, and which bug happened before which, probably doesn't matter.